.github/ISSUE_TEMPLATE/a11y-audit.md

```diff
@@ -0,0 +1,45 @@
+---
+name: A11y Audit
+about: For internal use only.
+title: "A11y Audit: "
+labels: a11y, 0 - new, p - high, needs triage
+assignees: ""
+---
+
+### Issue Summary
+
+<!-- A clear and concise description of the bug, defect, or problem -->
+
+### Related Standards
+
+<!-- [WCAG links, etc] -->
+
+### Reproduction Steps
+
+<!-- Steps to reproduce the behavior -->
+
+### Expected Behavior
+
+<!-- A clear and concise description of what you expected, or what should happen -->
+
+[Recommendation]
+
+[Code Example]
+
+### Remediation Steps
+
+<!-- Tips and approaches to remediating the issue, and bringing it into conformance. -->
+
+### Screenshots/Images
+
+<!-- If applicable, add screenshots to help define the bug or defect -->
```

Comment:
Aside from this one, are all other fields required? It's not clear from the comments.